django/contrib/admin/models.py

```diff
@@ -44,6 +52,52 @@ def log_action(
             change_message=change_message,
         )
 
+    def log_actions(
+        self, user_id, queryset, action_flag, change_message="", single_object=False
```

Comment:
Make `single_object` keyword-only:

```suggestion
        self, user_id, queryset, action_flag, change_message="", *, single_object=False
```

Reply:
Can you please elaborate a little?

Reply:
I don't understand why, when I suggested the following signature above:

```python
    def log_actions(self, user_id, queryset, action_flag, change_message="", *, single_object=False):
        ...
        if single_object and log_entry_list:
            ...
        ...
```

...you've gone for the following:

```python
    def log_actions(self, user_id, queryset, action_flag, change_message="", **kwargs):
        ...
        if kwargs.get("single_object", False) and log_entry_list:
            ...
        ...
```

This is less explicit and unnecessary.

Reply:
Got it, this was the first time I encountered the use of asterik(*) in the way mentioned by you, so I was a little skeptical about that. Updated the signature accordingly after going through the use once. Thank you for the suggestions Nick(@ngnpope )

Reply:
No worries! 🙂